src/components/Views/GradesView.jsx

```diff
@@ -15,12 +15,15 @@ import { PeriodSelector } from '../Atoms/Grades/PeriodSelector'
 import { PeriodSelectorButton } from '../Atoms/Grades/PeriodSelectorButton'
 import { YearSelectorButton } from '../Atoms/Grades/YearSelectorButton'
 import { TabTitle } from '../Atoms/TabTitle'
+import { AccountContext } from '../AppLayout'
 
 export const GradesView = () => {
   const { t } = useI18n()
   const { isMobile } = useBreakpoints()
 
-  const gradesQuery = buildGradesQuery()
+  const { currentAccount } = useContext(AccountContext)
```

Comment:
You can create a custom hook to extract the useContext like `useAccountContext`. It helps to refactor after

Reply:
It has been done here -> https://github.com/cozy/cozy-papillon-app/pull/29/commits/ca9dc8f080f372e312dc74c0ea9439cdef65a525